tools/provision/lib.sh

```diff
@@ -73,7 +73,11 @@ function install_rocksdb() {
         CLANG_INCLUDE="-I/usr/lib/clang/$CLANG_VERSION/include"
       fi
       pushd rocksdb-rocksdb-3.10.2
-      make static_lib CFLAGS="$CLANG_INCLUDE $CFLAGS"
+      if [[ $OS = "freebsd" ]]; then
+        CC=cc CXX=c++ gmake static_lib CFLAGS="$CLANG_INCLUDE $CFLAGS"
```

Comment:
@zi0r it's in the RocksDB ports Makefile: https://svnweb.freebsd.org/ports/head/databases/rocksdb/Makefile?view=markup&pathrev=362558#l41

The `${INSTALL_LIB}` will result in `install -s ...`, that will strip the symbols from the static library. :/ Can we change that to a `${INSTALL_DATA}`?